account/views.py

```diff
@@ -95,3 +95,12 @@ def post(self, request):
             user.save()
             return Response({"message": "Password Updated Successfully"}, status.HTTP_200_OK)
         return Response(serializer.errors, status.HTTP_400_BAD_REQUEST)
+
+
+class ProfileView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request):
+        user = request.user
```

Comment:
Let's check for what kind of user instance and use the appropriate serializer for each.